### DISCUSSION TITLE
device name under windows

### DISCUSSION THREAD

**Discussion by sudzwerg80** [OctoPrint/docker#212] [Q&A]
hello everyone iam trying to run the octoprint via docker for windows. i want to run it with webcam how i find the devicename under windows? i installed python for windows but that is not working because i get this error:
 `python -m serial.tools.miniterm
C:\Users\*****\AppData\Local\Programs\Python\Python310\python.exe: Error while finding module specification for 'serial.tools.miniterm' (ModuleNotFoundError: No module named 'serial.tools')` 
can someone please help me?

**Comment by ToyVo** [comment]
in case you haven't figured this out yet, in order to run that command, you'll need to install [pyserial](https://github.com/pyserial/pyserial/)
```sh
pip install pyserial
```
